app/Http/Controllers/ReportsController.php

```diff
@@ -1109,28 +1111,31 @@ public function getAssetAcceptanceReport($deleted = false) : View
         $this->authorize('reports.view');
         $showDeleted = $deleted == 'deleted';
 
-        /**
-         * Get all assets with pending checkout acceptances
-         */
-        if($showDeleted) {
-            $acceptances = CheckoutAcceptance::pending()->where('checkoutable_type', 'App\Models\Asset')->withTrashed()->with(['assignedTo' , 'checkoutable.assignedTo', 'checkoutable.model'])->get();
-        } else {
-            $acceptances = CheckoutAcceptance::pending()->where('checkoutable_type', 'App\Models\Asset')->with(['assignedTo' => function ($query) {
-                $query->withTrashed();
-            }, 'checkoutable.assignedTo', 'checkoutable.model'])->get();
+        $query = CheckoutAcceptance::pending()
+            ->where('checkoutable_type', 'App\Models\Asset')
+            ->with([
+                'checkoutable' => function (MorphTo $query) {
+                    $query->morphWith([
+                        AssetModel::class => ['model'],
+                        Company::class => ['company'],
+                        Asset::class => ['assignedTo'],
+                    ])->with('model.category');
```

Comment:
I haven't had a chance to check this out yet but want to note that `morphWith` is the [documented](https://laravel.com/docs/10.x/eloquent-relationships#nested-eager-loading-morphto-relationships) way to eager load morphTo relationships. It looks weird but we landed on that while pairing.

Reply:
I think this is similar to how we handle the searchable relations - https://snipe-it.readme.io/docs/architecture#models-searchability-relationships-and-query-scopes